SpatialGDK/Source/SpatialGDK/Private/Utils/OpUtils.cpp

```diff
@@ -22,6 +22,22 @@ void FindFirstOpOfType(const TArray<Worker_OpList*>& InOpLists, const Worker_OpT
 	}
 }
 
+void FindAllOpsOfType(const TArray<Worker_OpList*>& InOpLists, const Worker_OpType InOpType, TArray<Worker_Op*>& FoundOpArray)
```

Comment:
Nit: OutFoundOpArray

Reply:
I think it's an in/out param. I don't know what the unreal style guide says about those but I'd be happy to not prepend anything to it.

Reply:
Ally also has a version of this function which returns the found Ops as an array, but I wanted to avoid the extra memory allocations here. Happy to change the name.